pandas/_typing.py

```diff
@@ -90,18 +90,12 @@
     # Name "npt._ArrayLikeInt_co" is not defined  [name-defined]
     NumpySorter = Optional[npt._ArrayLikeInt_co]  # type: ignore[name-defined]
 
-    from typing import SupportsIndex
-
-    if sys.version_info >= (3, 10):
-        from typing import Concatenate  # pyright: ignore[reportUnusedImport]
-        from typing import ParamSpec
-        from typing import TypeGuard  # pyright: ignore[reportUnusedImport]
-    else:
-        from typing_extensions import (  # pyright: ignore[reportUnusedImport]
-            Concatenate,
-            ParamSpec,
-            TypeGuard,
-        )
+    from typing import (
+        ParamSpec,
+        SupportsIndex,
+    )
+    from typing import Concatenate  # pyright: ignore[reportUnusedImport]
+    from typing import TypeGuard  # pyright: ignore[reportUnusedImport]
```

Comment:
I think you can merge these into one statement, e.g.
```python
    from typing import (
        Concatenate, # pyright: ignore[reportUnusedImport]
        ParamSpec,
        SupportsIndex,
        TypeGuard, # pyright: ignore[reportUnusedImport]
    )
```

Reply:
Looks like isort is forcing these imports to be separate. Is that OK with you?

Reply:
That's fine.